packages/gatsby-theme-styleguide/src/gatsby-components/SiteMapData/tools.js

```diff
@@ -0,0 +1,69 @@
+/*
+basedir = '/src/docs' - using allFile
+basedir = 'docs' - using allMdx
+*/
+
+export const parseAllFiles = (list, { basedir = 'docs/', location } = {}) => {
```

Comment:
let's keep location inside the menu, components that will need it should compare it localy